bin/llvm-kompile

```diff
@@ -93,6 +94,10 @@ while [[ $# -gt 0 ]]; do
       python_cmd="$2"
       shift; shift
       ;;
+    --python-output-dir)
```

Comment:
Currently, `llvm-kompile --help` does not list this option.

Reply:
`pyk` calls `llvm-kompile` as

```
llvm-kompile pythonast --python /usr/bin/python3 -- -o /tmp/tmpr2d3kbct/_kllvm.cpython-38-x86_64-linux-gnu.so
```

If I understand correctly, this change will simplify the above call to

```
llvm-kompile pythonast --python /usr/bin/python3 --python-output-dir /tmp/tmpr2d3kbct
```

Reply:
Yes, that's correct. The value of `--python` will get called to generate the filename using the same module that I believe you use in Pyk.

I'll add the help message and merge.